packages/theme-default/src/components/Search/logic/providers/LocalProvider.ts

```diff
@@ -52,7 +52,7 @@ export class LocalProvider implements Provider {
     const searchIndexLang = lang ? `.${lang}` : '';
 
     const result = await fetch(
-      `${process.env.__ASSET_PREFIX__}/static/${SEARCH_INDEX_NAME}${searchIndexVersion}${searchIndexLang}.${searchIndexHash[searchIndexGroupID]}.json`,
+      `${__webpack_public_path__}static/${SEARCH_INDEX_NAME}${searchIndexVersion}${searchIndexLang}.${searchIndexHash[searchIndexGroupID]}.json`,
```

Comment:
Should we check if `__webpack_public_path__` ends with a slash?

Reply:
updated